MyBlog.Services/Areas/Users/AutoMapper/Dto/UserDtoGet.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace MyBlog.Service.Areas.Users.AutoMapper.Dto;
+
+public class UserDtoGet
+{
+    public int Id { get; set; }
+
+    public string FirstName { get; set; }
+
+    public string LastName { get; set; }
+
+    public string Password { get; set; }
+
+    public string Email { get; set; }
+
+    public string Phone { get; set; }
+
+    public DateTime DateCreateAccount { get; set; } = DateTime.Now;
+}
```

Comment:
i think, it`ll be nice, if you move DateTime initialization to UserDtoInput class, so, you create user and take actual date one time, but if you init this property in the UserDto class, every time you take User from Database, it will be different DateCreateAccount.